platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -85,6 +85,12 @@
         SELECTION_BUFFER,
     }
 
+    public enum MapRegionChangeState {
```

Comment:
Think this enum should be private? No need for it to be visible outside this class.